examples/cordova-sample/MyApp/README.md

```diff
@@ -3,18 +3,16 @@
 1. cd into the root of the example project, then:
 
 ```bash
-sh bin/setup.sh
-```
 
-2. api key
+sudo gem install bundler
```

Comment:
Petition to remove `sudo` from these directions 😇

Reply:
Or show `sudo` as an optional way and not recommended right from the start

Reply:
I'd sign that petition. Removing.

Reply:
✅